### DISCUSSION TITLE
Absolute Paths Always Linking To Default Locale

### DISCUSSION THREAD

**Discussion by fweth** [jlarmstrongiv/astro-i18n-aut#49] [General]
Do I get this right that I can't use global paths in hrefs (if they don't explicitly specify the locale), because they will always link to the default URL, so I need to use relative paths and double dot notation when I want to go "back"?  Or did I do something wrong in my configuration?

**Comment by jlarmstrongiv** [comment]
Unfortunately, URLs in hrefs are not automatically translated.  You will need to handle that yourself—whether you use `../sibling` or your own custom helpers.

I haven’t documented the built-in helpers yet (WIP), but try using `getLocaleUrl("/about", locale)`.  You can get the locale with `getLocale(Astro.url)`

A custom link component may make it easier too.  The reason the second argument of `getLocaleUrl` is not optional, is so that you can use these helpers in Astro, React, or your favorite component library/framework.
